tools/ou_period_statistics_joint.py

```
@@ -0,0 +1,322 @@
+        capture_output=True,
+        text=True,
+        check=False,
+    )
+    try:
+        parsed = ouv.parse_validation_metrics(completed.stdout)
+    except ValueError:
+        sys.stderr.write(completed.stdout[-4000:] + completed.stderr[-4000:])
+        raise SystemExit(
+            f"no VALIDATION_METRICS for {record.name} "
+            f"sigma_k={sigma_k} seed={seed}"
+        )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)